src/helpers.js

```diff
@@ -60,6 +60,9 @@ export function eachOperation(spec, cb, find) {
   // Iterate over the spec, collecting operations
   for (const pathName in paths) {
     for (const method in paths[pathName]) {
+      if (method.toUpperCase() === 'PARAMETERS') { 
```

Comment:
I see. The loop iterates the keys of each path, so "parameters" will appear. Doesn't matter if it appears before or after the "get".